src/Appwrite/Migration/Migration.php

```diff
@@ -91,6 +91,7 @@ abstract class Migration
         '1.5.10' => 'V20',
         '1.5.11' => 'V20',
         '1.6.0' => 'V21',
+        '1.6.1' => 'V21',
```

Comment:
I think the request & response filters also need an update?

Reply:
You mean creating/updating the existing filters? Or updating something to make sure the filters are used in 1.6.1?

Reply:
We don't need to update for the filters to apply for patch versions. See https://github.com/appwrite/appwrite/blob/8e021158286cd04b64b391afe86975e23368a52c/app/controllers/general.php#L605